cg/server/endpoints/orders.py

```diff
@@ -274,16 +275,18 @@ def validate_order(order_type: OrderType):
     response = {}
     if order_type == OrderType.BALSAMIC:
         response = balsamic_validation_service.validate(raw_order)
-    if order_type == OrderType.MICROBIAL_FASTQ:
+    elif order_type == OrderType.MICROBIAL_FASTQ:
         response = microbial_fastq_validation_service.validate(raw_order)
-    if order_type == OrderType.MICROSALT:
+    elif order_type == OrderType.MICROSALT:
         response = microsalt_validation_service.validate(raw_order)
-    if order_type == OrderType.MIP_DNA:
+    elif order_type == OrderType.MIP_DNA:
         response = mip_dna_validation_service.validate(raw_order)
-    if order_type == OrderType.SARS_COV_2:
+    elif order_type == OrderType.PACBIO_LONG_READ:
+        response = pacbio_long_read_validation_service.validate(raw_order)
+    elif order_type == OrderType.SARS_COV_2:
         response = mutant_validation_service.validate(raw_order)
-    if order_type == OrderType.RNAFUSION:
+    elif order_type == OrderType.RNAFUSION:
         response = rna_fusion_validation_service.validate(raw_order)
-    if order_type == OrderType.TOMTE:
+    elif order_type == OrderType.TOMTE:
         response = tomte_validation_service.validate(raw_order)
```

Comment:
can we make a map for this ? 🥲

Reply:
Would be much better! I think this endpoint will be around for when the customer saves cases/samples as well so it should definitely be done. 

I think when we created this it was intended to be a test endpoint that we would remove before going live, but I do not think that is the case anymore

Reply:
We should refactor this before we go live but not in this PR